src/components/tables/RowActions/AccessLinks/DisableDirective.tsx

```diff
@@ -73,7 +69,7 @@ function DisableDirective({
     return (
         <Progress
             progress={progress}
-            item={linkConfig.accessLink}
+            item={linkConfig.message}
```

Comment:
Initially I was trying to get everything shared. Then noticed that was a waste of time cause the new table view in the modal was just too different. However, I left some stuff the same as it kinda makes sense to just name more stuff similar and reduce the need for multiple types.